openapi.yml

```diff
@@ -231,6 +231,11 @@ paths:
           description: ID of the Workflow
           schema: {type: string}
           required: true
+      requestBody:
```

Comment:
True, we need a way to pass (OCR-D) parameters along with the workspace-id and the requestBody is the best place (in theory this could also be sent via URI encoded URL query params but that is ugly).